tgui/packages/tgui/interfaces/TurbineComputer.js

```diff
@@ -0,0 +1,64 @@
+import { useBackend } from '../backend';
+import { Button, LabeledList, Section } from '../components';
+import { Window } from '../layouts';
+
+export const TurbineComputer = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    compressor,
+    compressor_broken,
+    turbine,
+    turbine_broken
+  } = data;
+  const operational = Boolean(compressor && !compressor_broken && turbine && !turbine_broken);
+  return (
+    <Window>
+      <Window.Content>
+        <Section title="Status"
+          buttons={(
+            <>
+              <Button
+                icon={data.online ? 'power-off' : 'times'}
```

Comment:
for all cases of `data.VAR_NAME` you should just be able to assign it to a const instead of accessing `data` like I had you do with those other vars.

Reply:
Right. I forgot to take it all out. I will remove all those ones and commit them when I finish doing the split later today.